### PR TITLE
First app

### PR DESCRIPTION
**Pull request by abdul-mohsen**
completed the listing problem seamlessly with no problem. 

Right now the code just pop up a toast with message when the true or the false button being clicked.

### CODE REVIEW

**Comment by abdul-mohsen** [comment]
forgot to create another branch for the chapter 2 and it is now here

**Review by dgreenhalgh — CHANGES_REQUESTED** [review]
Great job with your code, but you have some merge conflicts that must be resolved before merging this pull request.

**Comment by abdul-mohsen** [comment]
I will check. But I did make a bug and I did not notice it until now (The two button true&false return true) and it is in all branches. I tried to fix it using git merge but it got worst because of the reverted commit. And now I have headache & no idea what going on :(( . So for the rest branches I decided to fix it by hand. Idk if there was a better solution for this mess XD

File: app/src/main/java/com/example/geoquiz/MainActivity.kt [hunk 1]
```diff
@@ -2,10 +2,67 @@ package com.example.geoquiz
...
+        nextButton = findViewById(R.id.next_button)
+        questionTextView = findViewById(R.id.question_text_view)
+
+        trueButton.setOnClickListener {view ->
+            checkAnswer(true)
+
+        }
+
+        falseButton.setOnClickListener {view ->
+            checkAnswer(true)
```
Comment (abdul-mohsen) [inline app/src/main/java/com/example/geoquiz/MainActivity.kt:40]: you can see it here

Comment (dgreenhalgh) [reply app/src/main/java/com/example/geoquiz/MainActivity.kt:40]: Ah, I see. Thanks for fixing this in future branches!

**Comment by dgreenhalgh** [comment]
I see that you have some merge conflicts on future branches, so I'll merge this to make those easier 😉

**Pull request merged**